components/collector/src/base_collectors/source_collector.py

```diff
@@ -217,13 +217,9 @@ async def __safely_parse_landing_url(self, responses: SourceResponses) -> URL:
     async def _landing_url(self, responses: SourceResponses) -> URL:
         """Return a user-friendly landing url.
 
-        Return the user supplied landing url parameter if there is one, otherwise translate the url parameter into
-        a default landing url.
+        Return the user supplied landing url parameter if there is one, otherwise return the API URL.
         """
-        if landing_url := self.__parameters.landing_url():
-            return landing_url
-        url = str(self.__parameters.api_url())
-        return URL(url.removesuffix("xml") + "html" if url.endswith(".xml") else url)
```

Comment:
Hah, looks like this was introduced to parse testdata only! https://github.com/ICTU/quality-time/pull/164/files

Reply:
I think it was to "help" the user not having to create a landing URL.